### ISSUE TITLE
A General Pain/Injury Buff

### ISSUE THREAD

**Issue by x0mek** [WW2-SS13/WW13#269]
As it is right now, injuries/getting shot/pain is almost completely unbalanced. It seems that there is no consistency in terms of where you get shot versus what effects are actually felt. You can take bullets to the face and then get up and keep fighting.

I think the stun/pain debuffs should be much much higher for getting shot. WWII wasn't call of duty, if you took a bullet ANYWHERE you were probably down for the count from just the pain and shock alone, not even mentioning critical areas like the head or chest. I think the current damage system could also be changed to be more dangerous, but that's another request for another day.

I think in general if we had more screaming people collapsing, unable to fight, than the current yakety-sax get shot > stand up > run around shooting, it would be a positive direction for the server. Would also give medics more to do.

TL;DR: Make pain more dangerous -> down for longer, more screaming, more movement penalties, make accuracy decrease almost exponentially. Now with the new stat system in the works I don't think there will be an issue simply lowering your accuracy stat by a lot if you are in pain.

**Comment by Eddyskingdom** [comment]
i think that is already done

**Comment by Eddyskingdom** [comment]
pain makes you see bad and your on the ground longer after a shot and head shot can ista kill
